bin/main.ml

```diff
@@ -179,22 +179,41 @@ let () =
     concat (prelude :: mra)
   in
 
-  let env     = Eval.build_evaluation_environment envinfo                      in
+  let tbl : (bytes, string list) Hashtbl.t = Hashtbl.create 10000 in
+  List.iter
+    (fun op -> Hashtbl.replace tbl op [])
+    List.(concat_map (fun x -> x.opcodes) @@ flatten blk_orded);
+  let tbl_update k f =
+    match Hashtbl.find_opt tbl k with
+    | Some x -> x
+    | None -> let x = f () in (Hashtbl.replace tbl k x; x)
+  in
+  (* Printf.printf "%d unique ops\n" Hashtbl.(length tbl); *)
+  (* flush stdout; *)
+
   Printexc.record_backtrace true;
+  let env =
+    match Eval.aarch64_evaluation_environment () with
```

Comment:
If we are using the default asli files now it would be good to simplify the command line arguments as well since they no longer work.

Reply:
done in https://github.com/UQ-PAC/gtirb-semantics/pull/5/commits/713b015280e4fcf8f6ae775b59794c928b6ea621